src/App/App.js

```diff
@@ -85,10 +86,10 @@ class App extends Component {
     } else {
       return (
         <main className='App'>
-          {/* {this.state.error && <Error error={this.state.error}/>} */}
           <Nav filterMovie={this.filterMovie} />
-          <img className="logo" src={logo} />
-          {this.state.randomMovie && <Banner randomMovie={this.state.randomMovie}/>}
+          <img className="logo" src={logo} alt="Logo image for Tangy Tomatillos with pink tomatillo icons"/>
```

Comment:
It won't let me leave this comment where I want to (componentDidMount) so I 'm putting it here - be sure to move your fetch into it's own file, per the rubric!